tests/components/deconz/test_climate.py

```diff
@@ -549,6 +549,120 @@ async def test_climate_device_with_fan_support(hass, aioclient_mock):
         )
 
 
+async def test_climate_device_SPZB0001(hass, aioclient_mock):
```

Comment:
The test doesn't follow the structure of the PR I mentioned. You might need to rebase your PR against dev to make it work.